report/test/renderer/element-screenshot-renderer-test.js

```diff
@@ -58,29 +58,13 @@ describe('ElementScreenshotRenderer', () => {
     };
     const el = ElementScreenshotRenderer.render(
       dom,
-      dom.document(),
       fullPageScreenshot,
       elementRectSC,
       renderContainerSizeDC
     );
 
     /* eslint-disable max-len */
-    expect(el.innerHTML).toMatchInlineSnapshot(`
-      "
-          <div class=\\"lh-element-screenshot__content\\" style=\\"top: -500px;\\">
-            <div class=\\"lh-element-screenshot__mask\\" style=\\"width: 500px; height: 500px; clip-path: url(#clip-0);\\">
-              <svg xmlns=\\"http://www.w3.org/2000/svg\\" height=\\"0\\" width=\\"0\\">
-                <defs>
-                  <clipPath clipPathUnits=\\"objectBoundingBox\\" id=\\"clip-0\\"><polygon points=\\"0,0             1,0            1,0.1          0,0.1\\"></polygon><polygon points=\\"0,0.7     1,0.7    1,1               0,1\\"></polygon><polygon points=\\"0,0.1        0.1,0.1 0.1,0.7 0,0.7\\"></polygon><polygon points=\\"0.5,0.1 1,0.1       1,0.7       0.5,0.7\\"></polygon></clipPath>
-                  <!-- clipPath filled by ElementScreenshotRenderer.renderClipPath -->
-                </defs>
-              </svg>
-            </div>
-            <div class=\\"lh-element-screenshot__image\\" style=\\"width: 500px; height: 500px; background-position-y: 0px; background-position-x: 0px; background-size: 1000px 1000px;\\"></div>
-            <div class=\\"lh-element-screenshot__element-marker\\" style=\\"width: 200px; height: 300px; left: 50px; top: 50px;\\"></div>
-          </div>
-        "
-    `);
+    expect(el.innerHTML).toMatchInlineSnapshot(`"<div class=\\"lh-element-screenshot__content\\" style=\\"top: -500px;\\"><div class=\\"lh-element-screenshot__mask\\" style=\\"width: 500px; height: 500px; clip-path: url(#clip-0);\\"><svg height=\\"0\\" width=\\"0\\"><defs><clipPath clipPathUnits=\\"objectBoundingBox\\" id=\\"clip-0\\"><polygon points=\\"0,0             1,0            1,0.1          0,0.1\\"></polygon><polygon points=\\"0,0.7     1,0.7    1,1               0,1\\"></polygon><polygon points=\\"0,0.1        0.1,0.1 0.1,0.7 0,0.7\\"></polygon><polygon points=\\"0.5,0.1 1,0.1       1,0.7       0.5,0.7\\"></polygon></clipPath></defs></svg></div><div class=\\"lh-element-screenshot__image\\" style=\\"width: 500px; height: 500px; background-position-y: 0px; background-position-x: 0px; background-size: 1000px 1000px;\\"></div><div class=\\"lh-element-screenshot__element-marker\\" style=\\"width: 200px; height: 300px; left: 50px; top: 50px;\\"></div></div>"`);
```

Comment:
insert some newlines to make this somewhat readable?

Reply:
insert where? the test or in generation?

Reply:
test